### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by YehorHeiko**
- [DEMO LINK](https://YehorHeiko.github.io/layout_catalog/)
- [TEST REPORT LINK](https://YehorHeiko.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,11 +2,370 @@
...
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+    content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet"
+    href="./styles/main.scss"
+    >
```
Comment (volodymyr-soltys97) [inline src/index.html:11]:
```suggestion
    <link rel="stylesheet" href="./styles/main.scss">
```

File: src/index.html [hunk 2]
```diff
@@ -2,11 +2,370 @@
...
+
+            <div class="card__reviews">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="card__reviews-label">Reviews: 5</div>
```
Comment (volodymyr-soltys97) [inline src/index.html:161]: Use the semantic tag for text everywhere

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews-label {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/card.scss:47]: Fix it everywhere
```suggestion
  &__reviews {
    display: flex;
    justify-content: space-between;
    
    &-label {
      display: flex;
      justify-content: center;
      align-items: center;
    }
  }
```

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,22 @@
...
+
 body {
   margin: 0;
+  font-family: Roboto, Cinzel, sans-serif;
+}
+
+html {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:20]: Default value, so you can just remove it
```suggestion
```

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,22 @@
...
+@import "blocks/stars";
+@import "blocks/container";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Cinzel, sans-serif;
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:14]: Move font family to the html selector

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job!
This part of the task is in complete
image

File: src/index.html [hunk 6]
```diff
@@ -2,11 +2,368 @@
...
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+              >
```
Comment (OleziO) [inline src/index.html:35]:
```suggestion
            <a
              href="#laptops&computers"
              class="nav__link"
              data-qa="nav-hover"
            >
```

File: src/styles/main.scss [hunk 7]
```diff
@@ -1,3 +1,22 @@
...
+@import './blocks/container';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+
+}
```
Comment (OleziO) [inline src/styles/main.scss:15]:
```suggestion
}
```

**Review by OleziO — CHANGES_REQUESTED** [review]
image
Make the effect of hovering over the button only when hovering over it, not over the entire card

**Comment by YehorHeiko** [comment]
Не обязательно при наведении на кнопку увеличивать карту. Этого нет в задании!

**Review by etojeDenys — CHANGES_REQUESTED** [review]
1. about hover effect:
image


2. try to fix your demo link
image

**Pull request closed without merging**